platform/ios/PolyPodApp/PolyPodUITests/SmokeTest.swift

```diff
@@ -0,0 +1,93 @@
+// Please remove this line and the empty one after it
+
+import XCTest
+
+class SmokeTest: XCTestCase {
```

Comment:
I think this would require a more specific name, like FeatureContainerSmokeTest, or something like that.

Reply:
Or just FeatureSmokeTest, since it's opening all the features? But I'm fine with any of those three.